From 31175bc96558c88101d50d625182f79f3cf72381 Mon Sep 17 00:00:00 2001
From: Jason Abele <jason@cozybit.com>
Date: Wed, 11 Jul 2012 16:53:14 -0700
Subject: [PATCH 5/5] Lock the openwrt feed versions

---
 feeds.conf.default |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/feeds.conf.default b/feeds.conf.default
index 518bf68..ae6af3a 100644
--- a/feeds.conf.default
+++ b/feeds.conf.default
@@ -1,6 +1,6 @@
-src-svn packages svn://svn.openwrt.org/openwrt/packages
-src-svn xwrt http://x-wrt.googlecode.com/svn/trunk/package
-src-svn luci http://svn.luci.subsignal.org/luci/trunk/contrib/package
+src-svn packages svn://svn.openwrt.org/openwrt/packages@32583
+#src-svn xwrt http://x-wrt.googlecode.com/svn/trunk/package
+src-svn luci http://svn.luci.subsignal.org/luci/trunk/contrib/package@8846
 #src-svn phone svn://svn.openwrt.org/openwrt/feeds/phone
 #src-svn efl svn://svn.openwrt.org/openwrt/feeds/efl
 #src-svn xorg svn://svn.openwrt.org/openwrt/feeds/xorg
-- 
1.7.5.4

